_index.md

```diff
@@ -0,0 +1,50 @@
+Here is the template we will use for the _index.md file
+---
+# Display name
+title: Nimanthi Yaseema
+
+# Username (this should match your github username)
+authors:
+- Nimanthi885
+
+# Role/position e.g. Graduate student
+
+
+# Organization membership
+Organizations/Affiliations:
+organizations:
+- name: University of Regina
+ 
+
+# A list of your interests, add as many as you want
+interests:
+- Algebraic Topology
+- General Purpose Data Science
+- Machine learning
+
+# Your educational history, add as many as you want
+education:
+  courses:
+  - course: MSc -  Mathematics
+    institution: Univerisity of Regina
+  - course: BSc - Mathematics
+    institution: Universidad of Peradeniya
+
+# Social/Academic Networking, delete any items you don't want to use
+social:
+- icon: github
+  icon_pack: fab
+  link: https://github.com/Nimanthi885
+#- icon: linkedin
+#  icon_pack: fab
+#  link: https://www.linkedin.com/in/nimanthi-yaseema-064863206/
+
+user_groups:
+- committee: members
+  role: ATCO
+
+tags:
+- '2021'
+---
+This is a freeform text area which supports markdown syntax. Fill in a couple of
```

Comment:
You probably want to remove these two lines or update them to say something about yourself.